source/_integrations/androidtv_remote.markdown

```diff
@@ -31,18 +31,21 @@ The entity has the `current_activity` attribute that shows the current foregroun
 You can use the `remote.turn_off`, `remote.turn_on`, `remote.toggle`,  and `remote.send_command` services from the [remote](/integrations/remote/) platform.
 
 For a list of the most common commands you can send to the Android TV via `remote.send_command` see: [TvKeys](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt).
-For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
+For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90) (note most of the keys in the full list typically don't work).
```

Comment:
We should not link to source code, this will scare end-users. Let's remove it and document what is needed.

Reply:
I removed it. The TvKeys.txt link above already links to the full list.

Reply:
The point being, that link shouldn't be there either.

Reply:
Done.